bin/pycbc_single_template

```diff
@@ -293,8 +295,19 @@ with ctx:
     parse_row.params = t()
     for param in row.fieldnames:
         setattr(parse_row.params, param, row[param][0])
-    chisq_bins = int(vetoes.SingleDetPowerChisq.parse_option(parse_row,
-                                                               opt.chisq_bins))
+
+    chisq_bins_float = vetoes.SingleDetPowerChisq.parse_option(parse_row,
+        opt.chisq_bins)
+    if chisq_bins_float.isnan() or (int(chisq_bins_float) < 
+                                    opt.minimum_chisq_bins): 
+        if opt.minimum_chisq_bins:
+            chisq_bins = opt.minimum_chisq_bins
+            logging.warning( "Number of chisq bins is less than minimum or is NaN."
+                            + (" Using %d bins." % opt.minimum_chisq_bins) )
+        else:
+            raise ValueError("Chisq bins is NaN and no default minimum set.")
```

Comment:
This could be reached by having a NaN _or_ if the formula evaluated to a negative number.  So, the error should say 'NaN or negative'